pandas/tests/generic/test_series.py

```diff
@@ -134,3 +134,17 @@ def finalize(self, other, method=None, **kwargs):
         # reset
         Series._metadata = _metadata
         Series.__finalize__ = _finalize  # FIXME: use monkeypatch
+
+    def test_series_clipping_with_na_values(self):
```

Comment:
move to pandas/tests/series/methods/test_clip.py

Reply:
Done